From 7ae77dea7960f912f9ea33992303c0c544c84838 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Sat, 24 Feb 2018 15:39:49 +0800
Subject: [PATCH] Revert "ARM: rockchip_defconfig: remove
 CONFIG_ANDROID_LOW_MEMORY_KILLER"

This reverts commit aa349fbf38defc84de8ab06900f0abbd927865ea.

Failed to pass vintf compatibility test.

Change-Id: I002feeb65cc2d7d679c0a00a984b149eef068be1
---
 arch/arm/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index fff86902ef62..014cf338454c 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -601,6 +601,7 @@ CONFIG_STAGING=y
 CONFIG_INV_MPU_IIO=y
 CONFIG_ASHMEM=y
 CONFIG_ANDROID_TIMED_GPIO=y
+CONFIG_ANDROID_LOW_MEMORY_KILLER=y
 CONFIG_SYNC=y
 CONFIG_SW_SYNC=y
 CONFIG_SW_SYNC_USER=y
-- 
2.35.3

